src/main/java/com/google/sps/servlets/AboutServlet.java

```diff
@@ -32,29 +33,41 @@
 
 @WebServlet(urlPatterns = "/about")
 public class AboutServlet extends HttpServlet {
+  private String staticResponse;
 
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    System.out.println("REQUEST AT: " + request.getServletPath());
-    response.setContentType("text/html;");
-
+  public void init() {
     JinjavaConfig config = new JinjavaConfig();
     Jinjava jinjava = new Jinjava(config);
     try {
       jinjava.setResourceLocator(
           new FileLocator(
               new File(this.getClass().getResource(ResourceConstants.TEMPLATES).toURI())));
-    } catch (URISyntaxException e) {
+    } catch (URISyntaxException | FileNotFoundException e) {
       System.err.println("templates dir not found!");
```

Comment:
In general make the errors as informative as possible. For eg. you know here which template you did not find, so print that. Same for all other errors (can be done in a separate PR). 

A notable exception is when dealing with PII. Your errors should, to the extent possible, not log PII.